src/main/java/uk/gov/digital/ho/hocs/casework/api/dto/SearchRequest.java

```diff
@@ -17,6 +21,11 @@
     private String reference;
 
     @JsonProperty("caseType")
+    @JsonSetter
+    private void caseTypeDeserialize(String caseType) {
+        this.caseTypes = Arrays.asList(caseType.split(","));
```

Comment:
Shockingly bad code. I expected much better from you.

Reply:
learnt from the best ;-)